README.md

```diff
@@ -27,10 +30,16 @@
 
 # RudderStack Unity SDK
 
-RudderStack's Unity SDK lets you track event data from your Android applications. After integrating the SDK, you will be able to send the event data to your preferred destination/s such as Google Analytics, Amplitude, and more.
+The RudderStack Unity SDK is a wrapper for the RudderStack [Android SDK](https://www.rudderstack.com/docs/sources/event-streams/sdks/rudderstack-android-sdk/) and [iOS SDK](https://www.rudderstack.com/docs/sources/event-streams/sdks/rudderstack-ios-sdk/). It is used for tracking game event data.
```

Comment:
This is a wrong statement. The v2 is no longer a wrapper of Android & iOS; it is a native SDK. We can mention the supported platforms here.

Reply:
Made the change.